shell/browser/electron_browser_client.cc

```diff
@@ -112,6 +112,9 @@
 #include "ui/native_theme/native_theme.h"
 #include "v8/include/v8.h"
 
+#include "chrome/browser/webauthn/authenticator_request_scheduler.h"
+#include "chrome/browser/webauthn/chrome_authenticator_request_delegate.h"
```

Comment:
Can you move the 2 lines to the list above? Headers generally are only listed separated when they are conditional.

Reply:
Move them up.